front/app/components/IdeaCards/IdeasWithoutFiltersSidebar.tsx

```diff
@@ -251,45 +235,30 @@ class IdeasWithoutFiltersSidebar extends PureComponent<
     return true;
   };
 
-  render() {
-    const { selectedView } = this.state;
-    const {
-      participationMethod,
-      participationContextId,
-      participationContextType,
-      defaultSortingMethod,
-      windowSize,
-      ideas,
-      className,
-      allowProjectsFilter,
-      showViewToggle,
-      project,
-    } = this.props;
-    const { queryParameters, list, hasMore, querying, loadingMore } = ideas;
-    const hasIdeas = !isNilOrError(list) && list.length > 0;
-    const locationEnabled = this.isFieldEnabled('location');
-    const topicsEnabled = this.isFieldEnabled('topic_ids');
-    const showViewButtons = !!(locationEnabled && showViewToggle);
-    const showListView =
-      !locationEnabled || (locationEnabled && selectedView === 'card');
-    const showMapView = locationEnabled && selectedView === 'map';
-    const smallerThanBigTablet = !!(
-      windowSize && windowSize <= viewportWidths.largeTablet
-    );
-    const smallerThanSmallTablet = !!(
-      windowSize && windowSize <= viewportWidths.smallTablet
-    );
-    const biggerThanSmallTablet = !!(
-      windowSize && windowSize >= viewportWidths.smallTablet
-    );
-    const biggerThanLargeTablet = !!(
-      windowSize && windowSize >= viewportWidths.largeTablet
-    );
-    const smallerThan1100px = !!(windowSize && windowSize <= 1100);
-    const smallerThanPhone = !!(
-      windowSize && windowSize <= viewportWidths.phone
-    );
-
+  const { list, hasMore, querying } = ideas;
+  const locationEnabled = isFieldEnabled('location');
+  const topicsEnabled = isFieldEnabled('topic_ids');
+  const showViewButtons = !!(locationEnabled && showViewToggle);
+  const showListView =
+    !locationEnabled || (locationEnabled && selectedView === 'card');
+  const showMapView = locationEnabled && selectedView === 'map';
+  const smallerThanBigTablet = !!(
+    windowSize && windowSize <= viewportWidths.largeTablet
+  );
+  const smallerThanSmallTablet = !!(
```

Comment:
Not blocking but could it be possible to use the new hook for this - `useBreakpoint`?